db/migrate/20220429112416_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :author, foregin_key: { to_table: 'users' }, index:true
```

Comment:
- [ ] You made a simple mistake while adding a foreign key. It should be` foreign_key` not `foregin_key`. That's why it is not showing in your schema. Please fix it. 
For that, you can roll back your database and edit this file and migrate again.
![image](https://user-images.githubusercontent.com/38635158/165939879-abf08d5e-1e95-49b1-bdf7-af5b1692b4de.png)